rebuild/src/functions.h

```diff
@@ -0,0 +1,12 @@
+#ifndef BLST_TS_FUNCTIONS_H__
+#define BLST_TS_FUNCTIONS_H__
+
+#include "addon.h"
+#include "blst.hpp"
+#include "napi.h"
+
+namespace Functions {
+void Init(const Napi::Env &env, Napi::Object &exports);
+}
```

Comment:
The `addon.h` used lowercase function names - is there a reason why it's inconsistent here? Some upstream constraints? If not, I'd recommend adding [Clang Format](https://clang.llvm.org/docs/ClangFormat.html) to the CI.

Reply:
The rest of the `Init` functions are static functions on the classes so i just matched the casing of those.  I suppose this should be lowercase as you mentioned though.  I also lowercased the namespace.  Updated.

```c++
namespace functions {
void init(const Napi::Env &env, Napi::Object &exports);
}
```